infinite_games/base/validator.py

```diff
@@ -532,12 +532,13 @@ def send_interval_data(self, miner_data):
                 )
                 if not res.status_code == 200:
                     bt.logging.warning(f'Error processing submission {res.content}')
+                    return False
                 else:
                     return True
-                time.sleep(1)
```

Comment:
not sure about the intention here. If the request was not successful, why do we return False instead of raising an error?
we could simply check `res.raise_for_status()` and then log the error in except

Reply:
@MadMedian to differentiate the other exceptions basically, although we can create exception, i think it is enough for now

Reply:
ok, let's log the errors where they happen, because it will be hard to follow.